multipass/Linux.md

```diff
@@ -18,7 +18,7 @@ This tutorial will give you an understanding of how Multipass works, and the ski
 
 <a href="#heading--install-multipass"><h2 id="heading--install-multipass">Install Multipass</h3></a>
 
-Multipass is available for Linux, macOs, or Windows. To install it on your OS of choice, please follow the instructions given [here](https://multipass.run/docs/how-to-guides). Note: This tutorial demonstrates use on Linux, specifically Ubuntu, but the experience on any OS should be similar.
+Multipass is available for Linux, macOs, and Windows. To install it on the OS of your choice, please follow the instructions given [here](https://multipass.run/docs/how-to-guides). Note that this tutorial demonstrates how to use Multipass on Linux, specifically Ubuntu, but the experience on any OS should be similar.
```

Comment:
Please incorporate a change that was suggested by another contributor: instead of:
> follow the instructions given [here]
substitute the generic "here" link with:
> follow the instructions given in the [how-to guides]